browser/ui/views/tabs/brave_tab_style_views.inc.cc

```diff
@@ -274,16 +274,17 @@ TabStyle::SeparatorBounds BraveVerticalTabStyle::GetSeparatorBounds(
 
   // Note: `leading` bounds are used for rect corner radius calculation and so
   // must be non-empty, even if we don't want to show it.
+  constexpr int extra_vertical_space = 8;
```

Comment:
Should the height be coming directly from `tab_style()->GetSeparatorSize()` above, rather than adjusted here?

That function gets its value from the `TAB_SEPARATOR_HEIGHT` layout value in `brave_layout_constants.cc`. Does it work to just adjust the value there?

Reply:
Wow, that's much better - I got fixated on the `extra_vertical_space` from looking at what the patch was doing. Thanks heaps @zenparsing